README.md

```diff
@@ -9,6 +9,7 @@ includes all changes in the latest node-statsd and many additional changes, incl
 * events
 * child clients
 * tcp protocol support
+* uds (Unix domain socket) protocol support
```

Comment:
I changed the protocol name from the other PR to be more in line with the other protocol names